src/Appointments.Console/Appointment Management System/Appointment Management System.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <RootNamespace>Appointment_Management_System</RootNamespace>
```

Comment:
Spaces in project files not recommended. Automated namespace creation reverted to snake casing which doesn't conform.